styles.css

```diff
@@ -249,6 +253,14 @@ nav ul {
   background: #ebf0ee;
 }
 
+.flex_container_secondary {
+  flex-direction: column;
+  align-items: center;
+  width: 95%;
+  padding: 20px;
+  gap: 20px;
+}
+
 .web_button {
```

Comment:
- [Optional] consider adding the hover and pressed state to all your buttons to match the figma template
![btn3](https://user-images.githubusercontent.com/94551484/206826619-54c8de0d-e3e6-4e17-9e67-b442a3c81065.png)